src-electron/db/db-mapping.js

```diff
@@ -590,6 +590,8 @@ exports.map = {
       commandRef: x.COMMAND_REF,
       incoming: dbApi.fromDbBool(x.INCOMING),
       outgoing: dbApi.fromDbBool(x.OUTGOING),
+      isIncoming: dbApi.fromDbBool(x.IS_INCOMING),
```

Comment:
What is the difference between IS_INCOMING and INCOMING, and how is an API consumer supposed to know?

Reply:
Let me give an overall picture. I was looking into creating an enabled only .zap file and noticed that when I do that I lost a bunch of commands from the clusters in the .zap file. The reason was because the way commands were stored in the .zap file. The client sourced commands were always stored within the client cluster block and the server sourced command was always stored in the server cluster block. What I had to do is separate those 2 entries into 2 rows i.e. incoming and outgoing had to be 2 separate rows in the endpoint_type_command table. The way they are separated, instead of `incoming` and `outgoing` you now have `isIncoming => 1 indicating incoming and 0 indicating outgoing` and `isEnabled=> 1 for enabled and 0 for disabled` . All the helper queries continue to work the same way by calculating incoming and outgoing using isIncoming and isEnabled to maintain backwards compatibility. Essentially now the endpoint_type_command table records whether the command is incoming or or outgoing and if it is enabled or not.
For future command helpers we can make better use of the isIncoming and isEnabled.